File: Java/Universidad.java
import java.util.ArrayList;
import java.util.List;

class Universidad {
    private String nombre;
    private List<Curso> cursos;

    public Universidad(String nombre) {
        this.nombre = nombre;
        this.cursos = new ArrayList<>();
    }

    public void agregarCurso(Curso curso) {
        this.cursos.add(curso);
    }

    public String getNombre() { return nombre; }
    public void setNombre(String nombre) { this.nombre = nombre; }
    public List<Curso> getCursos() { return cursos; }
    public void setCursos(List<Curso> cursos) { this.cursos = cursos; }

    @Override
    public String toString() {
        return "Universidad: " + nombre + " | Total cursos: " + cursos.size();
    }
}

File: Python/universidad.py
from typing import List

from curso import Curso


class Universidad:
    def __init__(self, nombre: str):
        self.nombre = nombre
        self.cursos: List[Curso] = []

    def agregar_curso(self, curso: Curso):
        self.cursos.append(curso)

    def get_nombre(self) -> str:
        return self.nombre

    def set_nombre(self, nombre: str):
        self.nombre = nombre

    def get_cursos(self) -> List[Curso]:
        return self.cursos

    def set_cursos(self, cursos: List[Curso]):
        self.cursos = cursos

    def __str__(self):
        return f"Universidad: {self.nombre} | Total cursos: {len(self.cursos)}"
